infrastructure/resource_group_level/modules/webapps/trade-finance-manager-api-calculated-variables.bicep

```diff
@@ -1,11 +1,12 @@
-// We need to add the connection strings separately beacause the value tfmUri comes from the TFM front-door, which would produce a circular dependency.
-// See the notes in trade-finance-manager-api-no-connection-strings.bicep
+// We need to add the calculated variables separately beacause the value tfmUri comes from the TFM front-door, which would produce a circular dependency.
+// See the notes in trade-finance-manager-api-no-calculated-variables.bicep
```

Comment:
You don't quite at the moment though. You set the variables earlier apart from the calculated ones, then replace them all with the combination of those and the calculated ones.